content/tutorials/presence.textile

```diff
@@ -101,30 +172,28 @@ blang[ruby].
     end
   ```
 
-  "See this step in Github":https://github.com/ably/tutorials/commit/presence-ruby-step2
-
 h2. Step 3 - Enter the presence set
 
-With Ably, you can be "attached":/realtime/channels to a channel (that is: be receiving messages published on that channel, or publishing them yourself) without necessarily being present on the channel. If you want to be present, and be seen by other users, you need to explicitly enter the presence set. For example, for a channel called "chatroom":
+With Ably, you can be "attached":/realtime/channels to a channel (that is: be receiving messages published on that channel, or publishing them yourself) without necessarily being present on the channel. If you want to be present, and be seen by other users, you need to explicitly enter the presence set. In this tutorial, you will create a channel called "chatroom" by attaching to it, and then enter the channel's presence set:
 
 blang[javascript,nodejs].
   ```[javascript]
-    var channel = realtime.channels.get('chatroom');
-    channel.attach(function(err) {
-      if(err) { return console.error("Error attaching to the channel"); }
-      console.log('We are now attached to the channel');
-      channel.presence.enter('hello', function(err) {
-        if(err) { return console.error("Error entering presence"); }
-        console.log('We are now successfully present');
+      // Attach to the "chatroom" channel
+      const channel = realtime.channels.get("chatroom");
+      await channel.attach((err) => {
+        if (err) {
+          return console.error("Error attaching to the channel.");
+        }
       });
-    });
-  ```
 
-blang[javascript].
-  "See this step in Github":https://github.com/ably/tutorials/commit/presence-javascript-step3
-
-blang[nodejs].
-  "See this step in Github":https://github.com/ably/tutorials/commit/presence-nodejs-step3
+      // Enter the presence set of the "chatroom" channel
+      await channel.presence.enter("hello", (err) => {
+        if (err) {
+          return console.error("Error entering presence set.");
+        }
```

Comment:
Just a suggestion, i like how you do this further down, (see line: 244)
`if (err) return console.error("Error entering presence set.");`

Reply:
Done